src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,20 +25,26 @@
   private String user;
   private String text;
   private long timestamp;
+  private String recipient;
 
   /**
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
```

Comment:
I know the docs you're following don't do this, but you should update the comments here to reflect the new variable you're adding.

Reply:
Updated the method comment to include recipient!